src/Analyzers/Core/CodeFixes/NamingStyle/NamingStyleCodeFixProvider.cs

```diff
@@ -30,7 +30,7 @@ namespace Microsoft.CodeAnalysis.CodeFixes.NamingStyles
     [ExportCodeFixProvider(LanguageNames.CSharp, LanguageNames.VisualBasic,
         Name = PredefinedCodeFixProviderNames.ApplyNamingStyle), Shared]
 #endif
-    internal class NamingStyleCodeFixProvider : CodeFixProvider
+    internal sealed class NamingStyleCodeFixProvider : CodeFixProvider
```

Comment:
generally sealing internal things to make it easier to reason about inheritance and overriding.